### PR TITLE
Phase 2 updating non Jakarta features for EE10 repeats.

### PR DESCRIPTION
**Pull request by jhanders34**
Add repeats for fats that test the following features:
- federatedRegistry-1.0
- jwt-1.0
- oauth-2.0
- openidConnectClient-1.0
- openidConnectServer-1.0
- restConnector-2.0
- samlWeb-2.0
- sessionCache-1.0
- socialLogin-1.0
- spnego-1.0
- webBundle-1.0

For: #18722

### CODE REVIEW

**Comment by jhanders34** [comment]
#libby #build

**Comment by jhanders34** [comment]
#libby #build

**Review by KyleAure — DISMISSED** [review]
sessionCache and restConnector look good.

**Review by ayoho — DISMISSED** [review]

**Review by jvanhill — DISMISSED** [review]
Security changes looked good.

**Comment by jhanders34** [comment]
#libby #build

**Review by c00crane — DISMISSED** [review]

**Comment by jhanders34** [comment]
#libby #build

**Comment by jhanders34** [comment]
#libby #build

File: dev/com.ibm.ws.security.oauth.oidc_fat.common/src/com/ibm/ws/security/oauth_oidc/fat/commonTest/CommonValidationTools.java [hunk 1]
```diff
@@ -1013,6 +1012,12 @@ public String getTokenFromResponse(Object response, String searchKey) throws Exc
                 } else {
                     for (String line : responseLines) {
                         if (line != null && line.contains(searchKey)) {
+							if (searchKey.charAt(searchKey.length() - 1) != ':') {
```
Comment (ayoho) [inline dev/com.ibm.ws.security.oauth.oidc_fat.common/src/com/ibm/ws/security/oauth_oidc/fat/commonTest/CommonValidationTools.java]: Any performance benefit to doing this as opposed to
```Java
if (!searchKey.endsWith(":")) {
```
?

**Review by ayoho — DISMISSED** [review]

File: dev/com.ibm.ws.security.oauth.oidc_fat.common/src/com/ibm/ws/security/oauth_oidc/fat/commonTest/CommonValidationTools.java (hunk 1 above)
Comment (ayoho) [reply dev/com.ibm.ws.security.oauth.oidc_fat.common/src/com/ibm/ws/security/oauth_oidc/fat/commonTest/CommonValidationTools.java]: (And replace the tabs with spaces.)

**Review by KyleAure — APPROVED** [review]
sessionCache and restConnector look good.

**Comment by jhanders34** [comment]
#libby

**Pull request merged**